dlt/normalize/validate.py

```diff
@@ -41,3 +45,13 @@ def verify_normalized_table(
             f"`{table['table_format']}` for table `{table['name']}`. "
             "The setting will probably be ignored."
         )
+
+    parent_key = get_first_column_name_with_prop(table, "parent_key")
+    if parent_key and not is_nested_table(table):
```

Comment:
we could consider cleaning up these kind of problems automatically, no?

Reply:
IMO in 90% this is user trying their own table structure...